app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks = 0
+    for index, friend in enumerate(friends):
```

Comment:
`index` variable is not used in the code, you could remove it and not use `enumerate`

Reply:
done